integration_tests/pages/refer/confirmPerson.ts

```diff
@@ -1,13 +1,22 @@
+import { referPaths } from '../../../server/paths'
 import { personUtils } from '../../../server/utils'
 import Page from '../page'
-import type { Person } from '@accredited-programmes/models'
+import type { Course, CourseOffering, Person } from '@accredited-programmes/models'
 
 export default class ConfirmPersonPage extends Page {
```

Comment:
If we've renamed the view here to `person/show`, should we rename the Page here? I quite like the integration tests referring to pages the users see (i.e. users would call the page "Confirm Person", but in the next commit we've called the new Page "NewReferralPage" - I wonder if that should be "FindPerson" page instead. Whichever approach we take, it'd be good to be consistent across pages.